course/meetings/eeb-asher-martin-2018-02-14.md

```diff
@@ -0,0 +1,122 @@
+# Table of Contents
+
+1.  [Notes from meeting <span class="timestamp-wrapper"><span class="timestamp">[2018-02-14 Wed]</span></span>](#org2ac5210)
+    1.  [Format RQM fall 2018](#org55a3d35)
+    2.  [Teaching setup](#org9e6dffd)
+    3.  [Content redesign](#org90ef1d8)
+    4.  [Misc](#org01d33cc)
+    5.  [Action items](#org93d9402)
+
+<a id="org2ac5210"></a>
+
+# Notes from meeting <span class="timestamp-wrapper"><span class="timestamp">[2018-02-14 Wed]</span></span>
+
+
+<a id="org55a3d35"></a>
+
+## Format RQM fall 2018
+
+-   New course name: EB313 Quantitative methods for ecology in R
+    -   3rd year course, still open for 4th year students (who would have priority
+        enrolment)
+    -   Prereqs:
+        -   Currently a stats intro class and a ecology class.
+        -   3rd year should have enough research experience.
+        -   Not a problem with lacking computational knowledge, we teach them from
+            the ground up.
+-   Too many instructors for the number of students last year. The students liked
+    it but it is not sustainable long term
+-   They expect more students this year. The course is capped at 30, maybe will
+    be 15-20?
+    -   With more students, expect that some will drop, so make sure that we still
+        have enough assignment points handed out before the drop deadline for them
+        to know how they stand in the class.
+        -   Also wait with the group work until the drop deadline has past, we don't
+            want them to drop out from the groups.
+    -   Typically there are around 20-25 4th year students in the research course.
+        Could be a good idea to schedule for fall and advertise for these students.
+        -   They could send emails about this during summer.
+-   The same room for Tue and Thu would be great. To deal with less IT issues.
+-   Have a formal presentation for the final project
+    -   Angle as an end of semester symposium for the students (since no one took up
+        the offer to submit officially this year, we will make a presentation a
+        lecture component for last year).
```

Comment:
Yes, I wish we had of made that a grading component... Hindsight is 20/20 I guess. :P